doc/tutorial_conftest.rst

```diff
@@ -0,0 +1,528 @@
+Using and understanding the basis of ``Ragger`` testing capabilities
+====================================================================
+
+``Ragger`` is mostly used as a testing framework for Ledger application. To ease
+the integration in such scenario, it includes pre-defined ``pytest`` fixtures and
```

Comment:
What are other usage scenarios for ragger ? Not sure we have to say it's "mostly" used as a testing framework... For now, it seems to me it is "only" used for testing, so...

Reply:
Ragger is at first a client framework, which happens to be well fitted for testing and provides capabilities in this direction. I don't want to corner it in this latter role only.